frontend/src/components/BakerDetails/BakerDetailsPoolContent.vue

```diff
@@ -109,7 +109,7 @@
 					</template>
 				</DetailsCard>
 				<DetailsCard>
-					<template #title>Baking commission</template>
+					<template #title>Validation commission</template>
```

Comment:
```suggestion
					<template #title>Block commission</template>
```
I guess I missed one.